sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -366,8 +366,8 @@ class ResolveSessionCatalog(
     case ShowTables(SessionCatalogAndNamespace(_, ns), pattern) =>
       assert(ns.nonEmpty)
       if (ns.length != 1) {
-          throw new AnalysisException(
-            s"The database name is not valid: ${ns.quoted}")
+        throw new AnalysisException(
+          s"The database name is not valid: ${ns.quoted}")
```

Comment:
(This is not related to this pr though) Can you remove the line break? `throw new AnalysisException(s"The database name is not valid: ${ns.quoted}")`

Reply:
Yea, removed the line break for all cases in this file.